src/utils/workspaceUtils.ts

```diff
@@ -9,8 +9,10 @@ export interface WorkspaceProject {
 
 export interface Agent {
   name: string;
-  path: string;
+  workspace: string;
```

Comment:
Maybe keeping a different `path` terminology here is better? Considering https://github.com/gptme/gptme/issues/584